### ISSUE TITLE
Convert the legend in the radial tree diagram to checkboxes

### ISSUE THREAD

**Issue by simongray** [kuhumcst/DanNet#113]
The look should be the same, but the actual widgets should be checkboxes. The existing behaviour should be kept, but extended to focus again too to allow keyboard users to perform the same action.

Furthermore, clicking a checkbox should permanently select _only_ that relation and deselect the other checkboxes (perhaps they appear as a coloured ring with no fill when unchecked?). When not all checkboxes are checked, checking additional ones will add those to the visible part of the diagram. Once all checkboxes are checked, the deselect event occurs again when selecting one.
